code/modules/antagonists/eldritch_cult/eldritch_transmutations.dm

```diff
@@ -177,16 +177,26 @@
 	var/mob/living/carbon/carbon_user = user
 	for(var/obj/item/living_heart/LH in atoms)
 
-		if(LH.target && LH.target.stat == DEAD)
+		if(LH.target?.stat)
```

Comment:
```suggestion
		if(LH.target && LH.target.stat != CONSCIOUS)
```
good practice to put the define there, otherwise we wouldn't need them

Reply:
i'm saying these to all the "head developers" out there (you know who you are)

Reply:
I hate it when mom and dad fight